# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/nsAccessibilityService.h
# Commit: 09de143215a1
# Full Hash: 09de143215a13a22dcb2cd53d4ed1dfb60480c82
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-09-30 02:48 +0300
# Regressor Bug: 1987945
# File Overlap Count: 2
# Overlapping Files: accessible/base/nsCoreUtils.cpp, accessible/tests/browser/relations/browser_anchor_positioning.js
# Description:
#   Revert "Bug 1987945: apply code formatting via Lando" for causing bustages at DocAccessible.cpp.
# ==============================================================================

diff -r 826eb4ac00dd -r 09de143215a1 accessible/base/nsAccessibilityService.h
--- a/accessible/base/nsAccessibilityService.h	Tue Sep 30 02:35:22 2025 +0300
+++ b/accessible/base/nsAccessibilityService.h	Tue Sep 30 02:48:50 2025 +0300
@@ -267,23 +267,6 @@
                                    int32_t aAppUnitsPerDevPixel);
 
   /**
-   * Notify accessibility that an anchor positioned frame is
-   * about to be removed. This gives us a chance to update cached relations
-   * before the reflow where we will lose references to the anchor and won't be
-   * able to refresh its accessible's cache.
-   */
-  void NotifyAnchorPositionedRemoved(mozilla::PresShell* aPresShell,
-                                     nsIFrame* aFrame);
-
-  /**
-   * Notify accessibility that an anchor frame is about to be removed. This
-   * gives us a chance to update cached relations before the reflow where the
-   * anchor will be lost and we won't be able to refresh the accessible cache of
-   * prior relations.
-   */
-  void NotifyAnchorRemoved(mozilla::PresShell* aPresShell, nsIFrame* aFrame);
-
-  /**
    * Notify accessibility that an element explicitly set for an attribute is
    * about to change. See dom::Element::ExplicitlySetAttrElement.
    */